src/sentry/api/serializers/models/project.py

```diff
@@ -170,6 +133,140 @@ def serialize(self, obj, attrs, user):
         return data
 
 
+class DetailedProjectSerializer(ProjectWithTeamSerializer):
+    OPTION_KEYS = frozenset([
+        'sentry:origins',
+        'sentry:resolve_age',
+        'sentry:scrub_data',
+        'sentry:scrub_defaults',
+        'sentry:safe_fields',
+        'sentry:sensitive_fields',
+        'sentry:csp_ignored_sources_defaults',
+        'sentry:csp_ignored_sources',
+        'sentry:default_environment',
+        'sentry:reprocessing_active',
+        'sentry:blacklisted_ips',
+        'feedback:branding',
+        'digests:mail:minimum_delay',
+        'digests:mail:maximum_delay',
+        'mail:subject_prefix',
+        'mail:subject_template',
+    ])
+
+    def get_attrs(self, item_list, user):
+        attrs = super(DetailedProjectSerializer, self).get_attrs(
```

Comment:
not sure if this can happen, but if item_list is empty the sql query will have a syntax error.